cblack.py

```diff
@@ -31,25 +31,24 @@ class CBlackModuleLoader(type(_real_pathfinder)):
     https://stupidpythonideas.blogspot.com/2015/06/hacking-python-without-hacking-python.html
   """
 
-  # all "black" folders contain the substring "/black-<version>"
-  # (e.g "/usr/local/lib/python3.8/site-packages/black-22.3.0-py3.8-linux-x86_64.egg")
-  _black_folder = "/black-%s" % __version__
-
-  # local installation of black folder don't necessarily need to contain the
-  # version, so the path includes '/black/'
-  _black_local_folder = "/black/"
+  _black_folder_patterns = (
```

Comment:
Paths on windows are usually represented with backslashes, so hardcoding the forward slash (even though it supported on Windows) will filter out the actual module names for black.